packages/shared/src/components/auth/IncompleteRegistrationModal.tsx

```diff
@@ -49,13 +49,14 @@ function IncompleteRegistrationModal({
       {...props}
       onRequestClose={onRequestClose}
       contentClassName="flex flex-col"
+      style={{ content: { overflow: 'hidden' } }}
     >
       <header className="flex relative justify-between items-center p-3 w-full border-b border-theme-divider-tertiary">
-        <span className="font-bold typo-body">Set your timezone</span>
+        <span className="ml-2 font-bold typo-body">Set your timezone</span>
```

Comment:
I think we are stilling missing the height of the header to be 50px not?

Reply:
Ah, nice catch. I totally forgot about which one to make it 50px. It is the height indeed.